Showcase-iOS/Dashboard/ViewModel/ShowcaseAppViewModel.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ShowcaseAppViewModel.swift
+//  Showcase-iOS
+//
+//  Created by Edward Mtshweni on 2018/05/03.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import Foundation
+
+class ShowcaseAppViewModel {
```

Comment:
I dont understand what the point of this class is ?

Reply:
Since the Interactor is the one that knows about the Entity. I created a ViewModel to be used in the View

Reply:
Still doesnt make any sense. Lets talk about it tomorrow and you can refactor this. I dont think this is needed doesnt make sense to me but maybe you can explain it to me more. Ill merge it and we can refactor if needed.